src/lib/Behat/Helper/UtilityContext.php

```diff
@@ -271,4 +271,25 @@ public function waitUntilElementDisappears(string $cssSelector, int $timeout): v
             throw new Exception(sprintf('Element with selector: %s did not disappear in %d seconds.', $cssSelector, $timeout));
         }
     }
+
+    /**
+     * Uploads file from location stored in 'files_path' to the disc on remote browser machine.
+     *
+     * @param string $localFileName
+     *
+     * @return string
+     */
+    public function uploadFileToRemoteSpace(string $localFileName): string
+    {
+        $localFile = sprintf('%s%s', $this->getMinkParameter('files_path'), $localFileName);
```

Comment:
If we need a file for testing purposes why can't you simply create a `zip` archive in the repository? I'm mostly concerned about the performance of this.